sources/util.ts

```diff
@@ -56,6 +56,16 @@ export function isHTMLLegendElement(
 	return isElement(node) && node.tagName === "LEGEND";
 }
 
+export function isHTMLSvgElement(node: Node | null): node is SVGElement {
+	return isElement(node) && node.tagName === "svg";
+}
+
+export function isHTMLSvgTitleElement(
```

Comment:
For consistency include the name of the interface the node is implementing.

```suggestion
export function isSVGTitleElement(
```